src/hooks/useArticleLinksNavigation.js

```diff
@@ -168,3 +172,14 @@ const findVisibleLinks = container => {
   }
   return visibleLinks
 }
+
+const isImageInGallery = (galleryItems, link) => {
+  const aElement = link.querySelector('a')
+  if (!aElement) {
+    return false
+  }
+
+  const href = aElement.getAttribute('href')
+  const fileName = href.split(':')[1]
```

Comment:
`makeLinkClickEvent` uses the same method here, what about reuse the function how it finds `fileName`?

Reply:
Yea why not, maybe something like below if we start from anchor (because anchor is not always guaranteed within `isImageInGallery`)

```
const getFileNameFromAnchorElement = (aElement) => {
  // file name example in href : /wiki/File:Holly_Christmas_card_from_NLI.jpg
  // split to match the api file name
  const href = aElement.getAttribute('href')
  return href.split(':')[1]
}
```

Reply:
sounds good 👍